### PR TITLE
tests: Fixes spelling

### PR DESCRIPTION
**Pull request by randall**

### CODE REVIEW

**Review by shinrich — DISMISSED** [review]

**Review by bneradt — CHANGES_REQUESTED** [review]
Looks good. Thanks for fixing the spelling.

File: tests/gold_tests/h2/httpbin.test.py [hunk 1]
```diff
@@ -83,7 +83,7 @@
 # Test Cases
 # ----
 
-# Test Case 0: Basic request and resposne
+# Test Case 0: Basic request and Response
```
Comment (bneradt) [inline tests/gold_tests/h2/httpbin.test.py:86]: Looks like this is still "Response" with a capital R. Was that supposed to be lower-cased?

Thanks,
Brian

**Review by bneradt — CHANGES_REQUESTED** [review]

File: tests/gold_tests/h2/httpbin.test.py (hunk 1 above)
Comment (randall) [reply tests/gold_tests/h2/httpbin.test.py]: You're absolutely right. Should be fixed in the latest version

**Review by bneradt — APPROVED** [review]
Looks good. Thanks Randall!

**Pull request merged**